doc/sphinx/install/windows.rst

```diff
@@ -0,0 +1,134 @@
+Windows
+=======
+
+These directions assume you are working on Windows 10 and you have installed
+`Microsoft Visual Studio <https://visualstudio.microsoft.com/>`_
+and the `Intel Fortran and C++ compilers <https://software.intel.com/en-us/parallel-studio-xe>`_.
+If you only want to use the python bindings then other compilers, including
+the Microsoft Visual C++ compiler, will likely work.  However,
+if you want to link to Abaqus or ANSYS through the :ref:`external interface <external>`
+then you will need a Fortran compiler, which seems to mean using the Intel
+Fortran compiler for most setups.  At that point you may was well use
+Intel C++ as well.
+
+When installing Visual Studio include the include "Desktop development with C++"
+and "Python development" options.
+
+When installing the Intel compilers use the default selected packages.
+
+You will also need to install the `boost library <https://www.boost.org/users/download/>`_.  Installing boost is easy as it's a header library -- just unzip the archive to a convenient location.  In this example we use ``C:\Program Files\boost`` but you can put it wherever you want.
+
+Additionally, these directions assume you have the Windows command line
+version of `git <https://git-scm.com/download/win>`_ installed.
+
```

Comment:
I guess but I'm not sure we need to get and install ninja, if so please add to the requirement list

Reply:
Nope, it comes with Visual Studio.